app/components/SongCard/index.js

```diff
@@ -0,0 +1,120 @@
+/**
+ *
+ * MusicCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Image, Typography, Button } from 'antd';
+import { PlayCircleTwoTone, StopTwoTone } from '@ant-design/icons';
+
+const { Paragraph, Title } = Typography;
+
+const Container = styled(Card)`
+  && {
+    max-width: 300px;
+    height: 450px;
```

Comment:
fixed height and width is usually recipe for disaster, what happens if the length of the data that we;re showing is longer?